README.md

```diff
@@ -1,12 +1,12 @@
-# Salesforce DataCloud JDBC Driver
+# Salesforce Data Cloud JDBC Driver
 
-With the Salesforce Data Cloud JDBC driver you can efficiently query millions of rows of data with low latency, and perform bulk data extractions.
-This driver is read-only, forward-only, and requires Java 8 or greater. It uses the new [Data Cloud Query API SQL syntax](https://developer.salesforce.com/docs/data/data-cloud-query-guide/references/dc-sql-reference/data-cloud-sql-context.html).
+This driver allows you to efficiently query large datasets in Salesforce Data Cloud with low latency and perform bulk data extractions. It's read-only, forward-only, and requires Java 8 or later. The driver uses the Data Cloud Query API SQL syntax (see [Data Cloud Query API SQL syntax](https://developer.salesforce.com/docs/data/data-cloud-query-guide/references/dc-sql-reference/data-cloud-sql-context.html) for details).
 
+## Getting Started
 
-## Getting started
+### Maven Dependency
 
-To add the driver to your project, add the following Maven dependency:
+To use the driver in your Maven project, add the following to your `pom.xml`:
```

Comment:
```suggestion
To use the driver in your Maven project, add the following dependency to your `pom.xml`:
```

not sure, reads more natural to me that way... but I am not a native speaker, up to you, Dralia...